content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,101 @@
+---
+title: استخدام مينيكيوب لإنشاء عنقود
+weight: 10
+description: |-
+  تعلم ما هو عنقود كوبيرنيتيس.
+  تعلم ما هو مينيكيوب.
+  أنشئ عنقود كوبيرنيتيس.
+---
+
+<!DOCTYPE html>
+
+<html lang="ar">
+
+<body>
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+        <h3>الأهداف</h3>
+                <ul>
+                    <li>اطّلع على مفهوم عنقود كوبيرنيتيس.</li>
+                    <li>تعرّف على ماهية مينيكيوب.</li>
+                    <li>أنشئ عنقود كوبيرنيتيس على جهاز الكمبيوتر الخاص بك.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>عناقيد كوبيرنيتيس</h3>
+                <p>
+                    <b>تنسق كوبيرنيتيس عنقودًا من الحواسيب المتاحة بشكل عالي والمتصلة لتعمل كوحدة واحدة.</b> التجريدات في كوبيرنيتيس تتيح لك نشر التطبيقات المعبأة في حاويات إلى عنقود دون ربطها بشكل خاص بالأجهزة الفردية. للاستفادة من هذا النموذج الجديد للنشر، يجب أن تكون التطبيقات معبأة بطريقة تفصلها عن المضيفين الفرديين: يجب أن تكون معبأة في حاويات. التطبيقات المعبأة في الحاويات أكثر مرونة وتوفرًا من النماذج السابقة للنشر، حيث كانت التطبيقات مثبتة مباشرة على أجهزة معينة كحزم متكاملة تكاملًا عميقًا مع المضيف. <b>كوبيرنيتيس يتمتع بالقدرة على توزيع وجدولة حاويات التطبيقات عبر العنقود بطريقة أكثر كفاءة.</b> كوبيرنيتيس هي منصة مفتوحة المصدر وجاهزة للإنتاج.
```

Comment:
```suggestion
                    <b>ينسق كوبيرنيتيس عنقودًا من الحواسيب المتاحة بشكل عالي والمتصلة لتعمل كوحدة واحدة.</b> التجريدات في كوبيرنيتيس تتيح لك نشر التطبيقات المعبأة في حاويات إلى عنقود دون ربطها بشكل خاص بالأجهزة الفردية. للاستفادة من هذا النموذج الجديد للنشر، يجب أن تكون التطبيقات معبأة بطريقة تفصلها عن المضيفين الفرديين: يجب أن تكون معبأة في حاويات. التطبيقات المعبأة في الحاويات أكثر مرونة وتوفرًا من النماذج السابقة للنشر، حيث كانت التطبيقات مثبتة مباشرة على أجهزة معينة كحزم متكاملة تكاملًا عميقًا مع المضيف. <b>كوبيرنيتيس يتمتع بالقدرة على توزيع وجدولة حاويات التطبيقات عبر العنقود بطريقة أكثر كفاءة.</b> كوبيرنيتيس هي منصة مفتوحة المصدر وجاهزة للإنتاج.
```
for HA check https://cloud-native.slack.com/archives/C02UFT8V256/p1715111597613069